test/e2e/wmco_test.go

```diff
@@ -54,3 +60,13 @@ func setupWMCOResources() error {
 	}
 	return nil
 }
+
+// testOperatorDeployed tests that the operator pod is running
+func testOperatorDeployed(t *testing.T) {
```

Comment:
```suggestion
func testOperatorDeployedWithoutPrivateKeySecret(t *testing.T) {
```

Reply:
I made the change for the test name, i am hesitant to change the function name as it does not actually test that the private key secret does not exist.